tests/components/mqtt/test_device_trigger.py

```diff
@@ -22,10 +22,14 @@
     async_get_device_automations,
     async_mock_service,
 )
-from tests.components.blueprint.conftest import stub_blueprint_populate  # noqa: F401
 from tests.typing import MqttMockHAClient, MqttMockHAClientGenerator, WebSocketGenerator
 
 
+@pytest.fixture(autouse=True, name="stub_blueprint_populate")
+def stub_blueprint_populate_autouse(stub_blueprint_populate):
+    """Stub copying the blueprints to the config folder."""
+
+
```

Comment:
If we have this in `conftest.py` is it still needed in the separate modules?

Reply:
It's not auto used in the integrations.